### DISCUSSION TITLE
I want to delete unnecessary IDs

### DISCUSSION THREAD

**Discussion by kazuki-o** [GrapesJS/grapesjs#3974] [Q&A] [answered]
Hi,
Thank you for the wonderful editor.
I want to remove the unwanted ID added to the HTML and export it.
Is there a way to prevent the unique ID given when editing?
![sample](https://user-images.githubusercontent.com/6177389/143221118-47adb1e8-50dd-4313-be7a-479ef46881c3.png)

**Comment by artf** [comment, marked as answer]
Try to check [getHtml API](https://grapesjs.com/docs/api/editor.html#gethtml). You should be able to use the `cleanId` option

**Reply by kazuki-o** [reply]
Thank you for your reply.
It was very helpful.
I will try it as soon as possible.
